features/shop/cart_with_different_taxes_for _different_zones.feature

```diff
@@ -0,0 +1,48 @@
+@ui-cart
+Feature: Cart with different tax rates for different time zones
+    In order to order products with different tax rate for different zones
+    As a Customer
+    I want to be aware of taxes applied on my order
+
+    Background:
+        Given that store is operating on the France channel
+        And default currency is "EUR"
+        And tax rate "EU VAT" with 23% rate belongs to "Taxable Goods" category for "EU" zone
+        And tax rate "No tax" with 0% rate belongs to "Taxable Goods" category for "Rest of World" zone
+        And tax rate "Low VAT" with 5% rate belongs to "Low-taxed Goods"
+        And there is user "john@example.com" identified by "password123"
+        And catalog has a product "PHP T-Shirt" priced at €100.00 with "Taxable Goods" as tax category
```

Comment:
I'd split this into 2 steps. (we have fast Behat now :D)

``And store has a product "Bananas" priced at €2``
``And "Bananas" tax category is "Food"``

Something like that, this way we can build the product and have more flexibility, without too much detail in a single step.